ReactiveFeedback/Feedback.swift

```diff
@@ -35,11 +36,24 @@ public struct Feedback<State, Event> {
             // NOTE: `observe(on:)` should be applied on the inner producers, so
             //       that cancellation due to state changes would be able to
             //       cancel outstanding events that have already been scheduled.
-            return transform(state)
+            return transform(state.map(\.0))
                 .flatMap(.latest) { effects($0).producer.observe(on: scheduler) }
         }
     }
 
+    public static func derivingEvents<U, Effect: SignalProducerConvertible>(
```

Comment:
Could we standardise on either static factory methods or initialisers, and depreciate the non-preferred form?

I don't have strong preference, and the former looks better as a DSL.